documentation/serenity-js.org/docs/web-testing/your-first-web-scenario.mdx

```diff
@@ -561,15 +562,15 @@ In particular:
 - the question about `itemNames()` retrieves the [text content](/api/web/class/Text) of those elements,
   and applies a simple transformation to [trim](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trim)
   space characters from both ends of the value,
-- the question about `itemCalled(name)` uses Serenity/JS [Page Element Query Language](/handbook/web-testing/page-element-query-language)
+- the question about `itemCalled(name)` uses Serenity/JS's [Page Element Query Language](/handbook/web-testing/page-element-query-language)
```

Comment:
Can we use `Serenity/JS Page Element Query Language` instead as per the [Google style guide](https://developers.google.com/style/possessives#product,-feature,-and-company-names)